as2/server/redemption.md

```diff
@@ -0,0 +1,15 @@
+# Redemption
+##### Messages pertaining to the redemption server.
+---
+### Join Server
+Sent to the penguin trying to log in to the redemption server to give needed player information. `rjs`
+
+|Argument|Type|Variadic?|Default
+|---|---|---|---
+|Redemeed Books|String|Yes|NULL
```

Comment:
Redeemed Books -> redeemedBooks
(or perhaps a better short name? I am unable to think of one)

The argument type should also better describe the actual string - ie Unsigned integer array delimited by ","